qdrant-landing/content/articles/fastembed.md

```diff
@@ -0,0 +1,217 @@
+---
+title: "FastEmbed: Better than SentenceTransformers, 50% faster "
+short_description: "Quantized models with ONNXRuntime for Local Support"
+description: "FastEmbed is a Python library engineered for speed, efficiency, and accuracy"
+social_preview_image: /articles_data/fastembed/social_preview.png
+preview_dir: /articles_data/fastembed/preview
+weight: -40
+author: Nirant Kasliwal
+author_link: https://nirantk.com/about/ 
+date: 2023-10-18T13:00:00+03:00
+draft: false
+keywords:
+  - vector search
+  - embedding models
+  - Flag Embedding
+  - OpenAI Ada
+  - quantized embedding model
+---
+
+Data Science and Machine Learning practitioners often find themselves navigating through a labyrinth of models, libraries, and frameworks. Which model to choose, what embedding size, how to approach tokenizing, these are just some questions you are faced with when starting your work. We understood how, for many data scientists, they wanted an easier and intuitive means to do their embedding work. This is why we built FastEmbed (docs: https://qdrant.github.io/fastembed/) —a Python library engineered for speed, efficiency, and above all, usability. We have created easy to use default workflows, handling the 80% use cases in NLP embedding.
+
+### Current State of Affairs for Generating Embeddings
```

Comment:
```suggestion
### Generating Embeddings - PyTorch, TensorFlow or ONNX?
```

Reply:
We don't want to compare very directly -- and this focusses on the frameworks more than what we do better.